src/components/main-panel/main-panel.styles.tsx

```diff
@@ -63,7 +63,7 @@ export const Wrapper = styled.div<IWrapperStyleProps>`
 `;
 
 export const EmptyDisplayWrapper = styled.div<IWrapperStyleProps>`
-    position: relative;
+    position: absolute;
```

Comment:
why does it have to be absolute positioned? this is causing the empty display to stretch to the full width, going underneath the side panel